porcelain/miner.go

```diff
@@ -295,6 +298,57 @@ func MinerGetOwnerAddress(ctx context.Context, plumbing mgoaAPI, minerAddr addre
 	return address.NewFromBytes(res[0])
 }
 
+// MinerGetSectorSize queries for the sector size of the given miner.
+func MinerGetSectorSize(ctx context.Context, plumbing mgoaAPI, minerAddr address.Address) (*types.BytesAmount, error) {
+	rets, err := plumbing.MessageQuery(ctx, address.Address{}, minerAddr, "getSectorSize")
+	if err != nil {
+		return nil, errors.Wrap(err, "'getSectorSize' query message failed")
+	}
+
+	methodSignature, err := plumbing.ActorGetSignature(ctx, minerAddr, "getSectorSize")
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to acquire 'getSectorSize' signature")
+	}
+
+	sectorSizeVal, err := abi.Deserialize(rets[0], methodSignature.Return[0])
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to deserialize returned value")
+	}
+
+	sectorSize, ok := sectorSizeVal.Val.(*types.BytesAmount)
+	if !ok {
+		return nil, errors.New("failed to convert returned ABI value")
+	}
+
+	return sectorSize, nil
+}
+
+// MinerGetLastCommittedSectorID queries for the id of the last sector committed
+// by the given miner.
+func MinerGetLastCommittedSectorID(ctx context.Context, plumbing mgoaAPI, minerAddr address.Address) (uint64, error) {
+	rets, err := plumbing.MessageQuery(ctx, address.Address{}, minerAddr, "getLastUsedSectorID")
+	if err != nil {
+		return 0, errors.Wrap(err, "'getLastUsedSectorID' query message failed")
+	}
+
+	methodSignature, err := plumbing.ActorGetSignature(ctx, minerAddr, "getLastUsedSectorID")
+	if err != nil {
+		return 0, errors.Wrap(err, "failed to acquire 'getLastUsedSectorID' signature")
+	}
+
+	lastUsedSectorIDVal, err := abi.Deserialize(rets[0], methodSignature.Return[0])
+	if err != nil {
+		return 0, errors.Wrap(err, "failed to deserialize returned value")
+	}
```

Comment:
These last 14 lines are essentially identical to the previous method except for a single method argument. It might be a good idea to extract it into a private method for any future porcelain methods that do the same thing. Alternatively, we could combine these two into a new method called `QueryAndDeserialize` or something.